site/app/controllers/grading/popup_refactor/RubricGraderController.php

```diff
@@ -0,0 +1,145 @@
+<?php
+
+/**
+ * ---------------------
+ *
+ * RubricGraderController.php
+ * First Created By: Nia Heermance
+ *
+ * This class's createMainRubricGradeablePage will eventually be called when a
+ * Rubric Gradeable's grading page is opened.
+ *
+ * Currently, to access the page associated with this class, enter URL:
+ *
+ *     /courses/{_semester}/{_course}/gradeable/{gradeable_id}/grading_beta[?more_stuff_here_is_okay]
+ *
+ * This class is also responsible for updating popup windows created.
+ *
+ * ---------------------
+ */
+
+# Namespace:
+namespace app\controllers\grading\popup_refactor;
+
+# Includes:
+use app\controllers\AbstractController;
+use Symfony\Component\Routing\Annotation\Route;
+use app\libraries\GradeableType;
+
+# Main Class:
+class RubricGraderController extends AbstractController {
+    # ---------------------------------
+
+    # Member Variables:
+
+    /**
+     * The current gradeable being graded.
+     */
+    private $gradeable;
+
+    /**
+     * By what ordering are we sorting by.
+     * Controls where next and prev arrows go.
+     */
+    private $sort_type;
+
+    /**
+     * For a given ordering, do we sort it ascending "ASC" or descending "DSC".
+     * Controls where next and prev arrows go.
+     */
+    private $sort_direction;
+
+    # ---------------------------------
+
+
+    # ---
+
+
+    # ---------------------------------
+
+    # Functions:
+
+    /**
+     * Creates the Rubric Grading page.
+     * @Route("/courses/{_semester}/{_course}/gradeable/{gradeable_id}/grading_beta/grade")
```

Comment:
I've added authentication in the next PR I am about to post. Since these pages are currently blank, I see no harm in not having the authentication yet.

Reply:
Yeah that makes sense, as long as its not forgotten